lib/rust/frp2/src/data.rs

```diff
@@ -0,0 +1,28 @@
+//! Type of data passed in FRP events.
+
+use crate::prelude::*;
+
+
+
+// ============
+// === Data ===
+// ============
+
+/// Trait for types that can be used as event data in the FRP system. The inherited bounds are as
+/// follow:
+/// - [`Debug`]: For debug purposes in order to be able to trace the data flow when needed.
+/// - [`Clone + 'static`]: Needed in case the data needs to be cloned if it is passed to a sample
+///   port.
+#[allow(missing_docs)]
```

Comment:
We require `Clone`. It is written in a strange way because it's a boxed clone. Rust doesn't allow for nicer implementation, so I've written these docs instead.

Reply:
I added comment about it.